### ISSUE TITLE
Add human readable interpretation of error messages

### ISSUE THREAD

**Issue by yjkimjunior** [polkadot-js/api#853]
It is a common complaint that comes up in the Substrate Technical channel e..g 
![Screen Shot 2019-04-23 at 4 04 24 PM](https://user-images.githubusercontent.com/10432070/56587267-84b4d000-65e1-11e9-9d71-60fbe3afffc6.png)

**Comment by jacogr** [comment]
1010: Invalid Transaction (3) is exactly as returned by the node. 

In this case 3 means CantPay - Which is a mapping to a specific error enum. There are a number of them. 

The 1010 and other codes we could map, but the actual enum name for those are already returned. The (3) is an additional enum for the specific type, which may or may not be there - in some cases the actual full text gets returns as part of the data. 

TL;DR yes, should improve, but sit in the realm of substrate to provide the number -> name mappings. Any attempt to re-create those are fraught with issues.

**Comment by jacogr** [comment]
Unless we have some additional information in metadata possibly? Calling @todr here, maybe he has some ideas that doesn’t involve us having to find and duplicate all the secondary enums in the middleware.
